src/ddl/fluview.sql

```diff
@@ -373,6 +388,7 @@ CREATE TABLE `flusurv` (
   `epiweek` int(11) NOT NULL,
   `location` varchar(32) NOT NULL,
   `lag` int(11) NOT NULL,
+  `season` char(7) DEFAULT NULL,
```

Comment:
Okay to add this between existing columns? I also did not mark `season` as a key since it is just a less-descriptive date label.

Reply:
Its ***probably*** ok, as long as columns are always referenced by name and not by order.  In practice, files in `ddl/` are not used outside of CI and unit/integration testing, so this is very unlikely to affect production at all.  To make changes to the live existing production database, you should add a file to do schema-altering operations to https://github.com/cmu-delphi/delphi-epidata/tree/dev/src/ddl/migrations .

Reply:
Yep, @melange396 will be best for this I think 👍